dockerfiles/remote-plugin-kubernetes-tooling-1.0.0/etc/docker.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+CMD=$1
+if [[ $1 = build ]]; then
+    CMD="bud"
+fi
+
+buildah $CMD $2 $3 $4 $5 $6 $7 $8 $9
```

Comment:
maybe it is better to use just `$@` ?

Reply:
It will require using `sed` to replace the argument in `$@` if I understand correctly.
Since I'm not good at `sed` I chose that approach.

Reply:
you should use `shift` 
```
shift
buildah $CMD $@
```

so the first argument is removed ($1)